lib/autoupdate/agent/config.go

```diff
@@ -67,12 +69,84 @@ type UpdateSpec struct {
 
 // UpdateStatus describes the status field in update.yaml.
 type UpdateStatus struct {
-	// ActiveVersion is the currently active Teleport version.
-	ActiveVersion string `yaml:"active_version"`
-	// BackupVersion is the last working version of Teleport.
-	BackupVersion string `yaml:"backup_version"`
-	// SkipVersion is the last reverted version of Teleport.
-	SkipVersion string `yaml:"skip_version,omitempty"`
+	// Active is the currently active revision of Teleport.
+	Active Revision `yaml:"active"`
+	// Backup is the last working revision of Teleport.
+	Backup *Revision `yaml:"backup,omitempty"`
+	// Skip is the skipped revision of Teleport.
```

Comment:
Can we add a little bit more context?
```suggestion
	// Skip is the skipped revision of Teleport.
	// Skipped revisions are not applied because they
	// are known to crash.
```